coalib/output/ConfWriter.py

```diff
@@ -13,10 +14,9 @@ def __init__(self,
                  key_value_delimiters=('=',),
                  comment_separators=('#',),
                  key_delimiters=(',', ' '),
-                 section_name_surroundings=None,
+                 section_name_surroundings=MappingProxyType({"[": "]"}),
```

Comment:
sorry I didn't mention this, is it possible to place this into an own commit? As this change doesn't really fit into the commit message :)